From e736d4a1d31d84a17f2f0969c3e5612252211abc Mon Sep 17 00:00:00 2001
From: Shaohua Li <shli@fb.com>
Date: Fri, 22 Jan 2016 16:40:02 -0800
Subject: [PATCH 11/24] md-cluster: delete useless code
Git-commit: fc2561ec0afcd89e2ec6f28a82a9219582844964
Patch-mainline: v4.5-rc3
References: fate#316335

page->index already considers node offset. The node_offset calculation
in write_sb_page is useless and confusion.

Cc: Goldwyn Rodrigues <rgoldwyn@suse.com>
Cc: NeilBrown <neilb@suse.com>
Acked-by: Guoqing Jiang <gqjiang@suse.com>
Signed-off-by: Shaohua Li <shli@fb.com>
Acked-by: Guoqing Jiang <gqjiang@suse.com>
---
 drivers/md/bitmap.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/md/bitmap.c b/drivers/md/bitmap.c
index 2b3c57a..af18582 100644
--- a/drivers/md/bitmap.c
+++ b/drivers/md/bitmap.c
@@ -197,10 +197,6 @@ static int write_sb_page(struct bitmap *bitmap, struct page *page, int wait)
 	struct block_device *bdev;
 	struct mddev *mddev = bitmap->mddev;
 	struct bitmap_storage *store = &bitmap->storage;
-	int node_offset = 0;
-
-	if (mddev_is_clustered(bitmap->mddev))
-		node_offset = bitmap->cluster_slot * store->file_pages;
 
 	while ((rdev = next_active_rdev(rdev, mddev)) != NULL) {
 		int size = PAGE_SIZE;
-- 
2.6.2

